setup/docker-compose.yml

```diff
@@ -31,6 +31,7 @@ services:
       - WAIT_HOSTS_TIMEOUT=120
       - PROJECT_DIR=api-server
       - JAR_NAME=api-server-fat.jar
+      - JAVA_OPTS=-javaagent:org.jacoco.agent-0.8.6-runtime.jar=destfile=target/jacoco.exec
```

Comment:
This option needs to go in only for the test environment

Reply:
Made changes